Makefile

```diff
@@ -154,7 +154,7 @@ install-target: make-install-dirs
 	install -p -m 644 data/exaile.desktop \
 		$(DESTDIR)$(DATADIR)/applications/
 	install -p -m 644 data/exaile.appdata.xml \
```

Comment:
Not super familiar with the new specification (... why is there a new specification >_>), but looks like this should have a different filename too? metainfo.xml instead of appdata.xml

Reply:
Seems not to be necessary according to https://wiki.debian.org/AppStream/Guidelines
The debian lintian message is https://lintian.debian.org/tags/appstream-metadata-in-legacy-location